src/views/jarQuickPickItem.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { QuickPickItem } from "vscode";
+
+export class JarQuickPickItem implements QuickPickItem {
```

Comment:
The field `uri`, `type` are all public, and the class has no member method. It's not necessary to implements `QuickPickItem` to be a class. Please just extends it, something like:

```js
interface IJavaQuickPickItem extends QuickPickItem  {
    uri?: string;
    ...
}

```

When you want to use it. You can simply create a object:

```js
const jarItems: IJavaQuickPickItem  = {
    label: ...
    ...
}
```